packages/api/src/routes/medical/internal-facility.ts

```diff
@@ -32,7 +33,10 @@ router.put(
   "/",
   requestLogger,
   asyncHandler(async (req: Request, res: Response) => {
+    if (Config.isSandbox()) return res.sendStatus(httpStatus.NOT_IMPLEMENTED);
     const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    // TODO Thomas, do we need this?
+    // const skipHie = getFromQueryAsBoolean("skipHie", req);
```

Comment:
@thomasyopes 

From what I understood, this is a nice to have from [this PR](https://github.com/metriport/metriport/pull/2974), so maybe we can remove this and later create a new PR with those?